config/configuration.rb

```diff
@@ -0,0 +1,132 @@
+require 'pathname'
+require 'dotenv'
+
+# job composer app specific configuration
+class Configuration
+
+  # The app's configuration root directory
+  # @return [Pathname] path to configuration root
+  def config_root
+    Pathname.new(ENV["OOD_APP_CONFIG_ROOT"] || "/etc/ood/config/apps/myjobs")
+  end
+
+  def initializers_root
+    Pathname.new(ENV["OOD_APP_INITIALIZERS_ROOT"] || config_root.join("initializers"))
+  end
+
+  def load_external_config?
+    to_bool(ENV.fetch('OOD_LOAD_EXTERNAL_CONFIG', (rails_env == 'production')))
+  end
+
+  # Custom job templates source directory in /etc/.../myjobs/templates
+  # Defaults to APPROOT/templates if that directory exists or if
+  # not in production.
+  #
+  # @return [Pathname] path to configuration root
+  def templates_path
+    default = app_root.join('templates')
+
+    if (! default.directory?) && load_external_config?
+      config_root.join("templates")
+    else
+      default
+    end
+  end
+
+  # Load the dotenv local files first, then the /etc dotenv files and
+  # the .env and .env.production or .env.development files.
+  #
+  # Doing this in two separate loads means OOD_APP_CONFIG_ROOT can be specified in
+  # the .env.local file, which will specify where to look for the /etc dotenv
+  # files. The default for OOD_APP_CONFIG_ROOT is /etc/ood/config/apps/myjobs and
+  # both .env and .env.production will be searched for there.
+  def load_dotenv_files
+    # .env.local first, so it can override OOD_APP_CONFIG_ROOT
+    Dotenv.load(*dotenv_local_files) unless dotenv_local_files.empty?
+
+    # load the rest of the dotenv files
+    Dotenv.load(*dotenv_files)
+  end
+
+  def app_development_enabled?
+    return @app_development_enabled if defined? @app_development_enabled
+    to_bool(ENV.fetch('OOD_APP_DEVELOPMENT', DevRouter.base_path.exist?))
+  end
+  alias_method :app_development_enabled, :app_development_enabled?
+
+  def app_sharing_enabled?
+    return @app_sharing_enabled if defined? @app_sharing_enabled
+    @app_sharing_enabled = to_bool(ENV['OOD_APP_SHARING'])
+  end
+  alias_method :app_sharing_enabled, :app_sharing_enabled?
+
+  def brand_bg_color
+    ENV.values_at('OOD_BRAND_BG_COLOR', 'BOOTSTRAP_NAVBAR_DEFAULT_BG', 'BOOTSTRAP_NAVBAR_INVERSE_BG').compact.first
+  end
+
+  def brand_link_active_bg_color
```

Comment:
We do not include the `<style></style>` override so this configuration option is unnecessary.

Reply:
removed.